irohad/main/impl/block_loader_init.cpp

```diff
@@ -26,26 +26,26 @@ using namespace iroha::network;
 auto BlockLoaderInit::createService(
     std::shared_ptr<BlockQueryFactory> block_query_factory,
     std::shared_ptr<consensus::ConsensusResultCache> consensus_result_cache) {
-  return std::make_shared<BlockLoaderService>(block_query_factory,
-                                              std::move(consensus_result_cache));
+  return std::make_shared<BlockLoaderService>(
+      block_query_factory, std::move(consensus_result_cache));
 }
 
 auto BlockLoaderInit::createLoader(
     std::shared_ptr<PeerQueryFactory> peer_query_factory,
     std::shared_ptr<BlockQueryFactory> block_query_factory) {
   shared_model::proto::ProtoBlockFactory factory(
-      std::make_unique<shared_model::validation::BlockVariantValidator>());
-  return std::make_shared<BlockLoaderImpl>(peer_query_factory,
-                                           block_query_factory,
-                                           std::move(factory));
+      std::make_unique<
+          shared_model::validation::DefaultSignedBlockValidator>());
```

Comment:
https://github.com/hyperledger/iroha/blob/8883ac9000730229cd49a67c18bd993f2293fa08/shared_model/validators/default_validator.hpp#L108-L120

While it seems OK not to check the transaction signatures, because it will be checked later that the supermajority of peers have signed the block, and therefore transaction signatures should not be forged, a comment can be useful here.